src/Sylius/Bundle/GridBundle/Tests/DependencyInjection/SyliusGridExtensionTest.php

```diff
@@ -83,15 +83,15 @@ public function it_sets_filter_templates_as_parameters(): void
         $this->load([
             'templates' => [
                 'filter' => [
-                    'string' => 'AppBundle:Grid/Filter:string.html.twig',
-                    'date' => 'AppBundle:Grid/Filter:date.html.twig',
+                    'string' => 'Grid/Filter/string.html.twig',
```

Comment:
It's a test, but I guess there's `@` missing in the beginning.

Reply:
It was on purpose, you can't use @App. Not totally sure if it's the correct value.

Reply:
Ah yeah, you're right! :)